tests/app/data/test_schema_validation.py

```diff
@@ -83,8 +75,6 @@ def validate_json_against_schema(file, json_to_validate, schema):
             return []
         except jsonschema.exceptions.ValidationError as e:
             return ["Schema Validation Error! File [{}] does not validate against schema. Error [{}]".format(file, e)]
-        except JSONDecodeError as e:
```

Comment:
Okay, I see, yeah in that case I guess it makes sense. I'll add a handler to trap the error so that at least other schemas get parsed under the same test.